src/components/picker/PickerDialog.android.js

```diff
@@ -68,17 +68,6 @@ class PickerDialog extends BaseComponent {
     );
   }
 
-  renderPicker() {
-    const {children, onValueChange, selectedValue, renderNativePicker, wheelPickerProps, testID} = this.props;
-    if (_.isFunction(renderNativePicker)) {
-      return renderNativePicker(this.props);
-    }
-    return (
-      <WheelPicker selectedValue={selectedValue} onChange={onValueChange} {...wheelPickerProps} testID={`${testID}.wheelPicker`}>
```

Comment:
You don't need the `selectedValue`, `onValueChange`, and `wheelPickerProps` in the propTypes anymore :)

Reply:
Thanks, fixed